src/gui/optionsdlg.cpp

```diff
@@ -1182,6 +1206,9 @@ bool OptionsDialog::useAdditionDialog() const
 void OptionsDialog::enableApplyButton()
 {
     applyButton->setEnabled(true);
+
+    // Verify if the share action button must be enabled
+    m_ui->comboRatioLimitAct->setEnabled(m_ui->checkMaxRatio->isChecked() || m_ui->checkMaxSeedingMinutes->isChecked());
```

Comment:
I doubt this will be accepted here. Maybe it's better to make a new method with it and connect it with the checxkboxes somewhere in the constructor?

```
connect(m_ui->checkMaxRatio, SIGNAL(toggled(bool)), this, SLOT(toggleComboRatioLimitAct()));
connect(m_ui->checkMaxSeedingMinutes, SIGNAL(toggled(bool)), this, SLOT(toggleComboRatioLimitAct()));

void OptionsDialog::toggleComboRatioLimitAct()
{
    m_ui->comboRatioLimitAct->setEnabled(m_ui->checkMaxRatio->isChecked() || m_ui->checkMaxSeedingMinutes->isChecked());
}
```